magma/wireable.py

```diff
@@ -0,0 +1,19 @@
+from .array import Array
+from .tuple import Tuple
+from .protocol_type import magma_type
```

Comment:
nit (since new file :) )

```suggestion
from magma.array import Array
from magma.tuple import Tuple
from magma.protocol_type import magma_type
```